packages/gatsby/src/query/index.ts

```diff
@@ -145,17 +199,23 @@ function createStaticQueryJob(state, queryId) {
   }
 }
 
-function onDevelopStaticQueryDone({ job, result }) {
+function onDevelopStaticQueryDone({
+  job,
+  result,
+}: {
+  job: IQueryJob
+  result: IStaticQueryResult["result"]
+}): void {
   websocketManager.emitStaticQueryData({
     result,
-    id: job.hash,
+    id: job.hash as string,
```

Comment:
`as string` seems fishy here - shouldn't this be already typed as string? I do wonder if it might mean we have some type weirdness (`number` vs `string` somewhere)?

Reply:
It's `string | undefined` so ideally we'd check existence of `job.hash` before using it